autosklearn/pipeline/components/feature_preprocessing/nystroem_sampler.py

```diff
@@ -51,10 +53,11 @@ def fit(self, X, Y=None):
         if self.kernel == "chi2":
             if scipy.sparse.issparse(X):
                 X.data[X.data < 0] = 0.0
+                X = X.todense()
```

Comment:
Is this required? Sklearn don't give anything on whether it accepts sparse or not :/

Reply:
i fixed this is no longer included in the current PR